### DISCUSSION TITLE
【WTF Grant】WTF Academy 回溯性资助1

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#482] [General]
WTF Academy 的运行步入正轨，我们收到了来自 Ethereum Foundation ESP，StarkWare，和 Optimism的资助，并成功举办了 ETH Beijing Hackathon。今天，我们推出 WTF Academy 资助计划，用于奖励围绕 WTF Academy 建设的项目和 WTF 贡献者，包括但不限于开源教程和软件。

第一批入选的项目（共 $27,500）:
WTF Solidity: $7500
WTF Academy Education Platform: $7500 **已分配**
WTF Ethers: $2500
WTF Starknet: $2000 **已分配**
SoLive (Lead by @chongqiangchen): $2000 **已分配**
WTF Academy Quiz: $2000 **已分配**
WTF Academy 教学视频系列 (VerySmallWoods): $1000 **已分配**
WTF JavaScript: $1000 **已分配**
WTF Cairo: $1000 **已分配**
WTF Gas Optimization （Lead by @0xKaso ）: $500 **已分配**
WTF CTF（Lead by @flyq）: $500 **已分配**

每个项目的分配明细将以留言方式回复在这个帖子下面。如果有遗漏的项目，欢迎留言。

**Comment by flyq** [comment]
WTF CTF 的分配明细：
@flyq $300
@Big-AAron $200

**Reply by AmazingAng** [reply]
已转

**Comment by seasidejuvenile666** [comment]
WTF Academy Quiz的分配明细：
@AmazingAng $330
@0xc25fee20 $330
@wyf-ACCEPT $330
@seasidejuvenile666 $385
@JustinAsdz $385
@vecpeng $110
@cheng521521 $110

**Reply by AmazingAng** [reply]
已分配

**Comment by 0xKaso** [comment]
WTF Gas Optimization 的分配明细
@0xKaso $323.5
@AmazingAng $147.5
@simpleth $29

**Reply by AmazingAng** [reply]
已分配

**Comment by AmazingAng** [comment]
wtf.academy 教育平台分配明细：
@AmazingAng  $1750 
@0xDjangoMo $1500 
@chongqiangchen $1500
@stoneworld $1000 
@reborn-sama $1000 
@tankxu $500 
@jujube109 $250

**Reply by AmazingAng** [reply]
已发

**Comment by dxiongya** [comment]
solive solidity轻量级编辑器分配：
@chongqiangchen $700
@Glacier-Luo $700
@KieSun $266
@chiguayeshao $166
@0xDjangoMo $166

**Reply by AmazingAng** [reply]
已发

**Comment by seasidejuvenile666** [comment]
WTF JavaScript分配明细：
@lesenelir $200
@seasidejuvenile666 $200
@0xb00lean $100
@shengxinjing $200
@holazz $300

**Reply by AmazingAng** [reply]
已发
